azure-pipelines.yaml

```diff
@@ -41,23 +41,23 @@ parameters:
         timeForRed: '7 days'
         pipelineName: "Azure AAD group cleanup"
         branchName: "refs/heads/master"
-      - project: 'cnp'
-        definitionId: '545'
+      - project: 'PlatformOperations'
+        definitionId: '781'
         timeForAmber: '3 days'
         timeForRed: '7 days'
         pipelineName: "OWASP DB update"
         branchName: "refs/heads/master"
-      - project: 'cnp'
-        definitionId: '175'
+      - project: 'PlatformOperations'
+        definitionId: '772'
         timeForAmber: '3 days'
         timeForRed: '7 days'
         pipelineName: "ACR Base importer"
         branchName: "refs/heads/master"
       - project: 'Platform%20Engineering'
-        definitionId: '224'
+        definitionId: '756'
         timeForAmber: '3 days'
         timeForRed: '5 days'
-        pipelineName: "Preview scheduled cleanup"
+        pipelineName: "AKS Preview Helm Release Cleanup job"
```

Comment:
```suggestion
        pipelineName: "AKS Preview Helm Release Scheduled Cleanup job"
```

Reply:
I dont see that pipeline name in ADO currently?